docs/source/releases/v1_12_2a0.rst

```diff
@@ -23,37 +25,100 @@ Version 1.12.2a0 (2024-02-21)
   * Remove inconsistent unit test test_log_interactive_non_geoips
   * Modify algorithm_interpolator_colormapper to only provide input xarray
   * Re-write Dockerfile to restore functionality (installing geoips/passing tests)
+  * Fix product source_name duplicate entries bug
 * Refactor
 
   * Make JSON Plugin Registries Readable
   * Modify create_plugin_registries to use argparse instead of sys.argv
+* Testing Updates
 
-Enhancements
-============
+  * Added all_product_families.yaml to test all available product families
+* Breaking Changes
 
-Added Log with Emphasis function
---------------------------------
+  * Updated GeoIPS CLI to fix breaking changes introduced by all_product_families
```

Comment:
Do not merge until #444 and #465 have both been merged into main. Once we merge this, we can close PR #433.